src/main.rs

```diff
@@ -119,17 +122,49 @@ fn main() {
     let error_listen_for_ctrlc = Arc::new(Mutex::new(errors));
     let errors_for_rayon = error_listen_for_ctrlc.clone();
     let errors_final = error_listen_for_ctrlc.clone();
+    let is_in_listing = Arc::new(AtomicBool::new(false));
+    let cloned_is_in_listing = Arc::clone(&is_in_listing);
 
     ctrlc::set_handler(move || {
         error_listen_for_ctrlc.lock().unwrap().abort = true;
         println!("\nAborting");
+        if cloned_is_in_listing.load(Ordering::Relaxed) {
```

Comment:
I'm not sure we even need this boolean. If we detect a ctrl-c we should just abort irrespective of what `cloned_is_in_listin` is. 

How about we remove the 'if' but keep the process::exit(1) ?

Reply:
Do you think  'Aborting' should be eprintln instead of println? What do you think ?

Reply:
I added the boolean check because if we just use process::exit(1), I'm not sure what would happen when we are in the Rayon threads. Nevertheless, I can remove it if needed.

For me, I think it's nice to have "Aborting" in the console to show that dust has correctly intercepted the signal.